src/cobalt.rs

```diff
@@ -33,6 +33,31 @@ fn compare_paths(a: &Path, b: &Path) -> bool {
     }
 }
 
+/// Checks if one path is the starting point of another path.
+/// The tests are taken from tests/fixtures/`posts_in_subfolder`/
+#[test]
+fn test_starts_with_path() {
+    let posts_folder = Path::new("tests/fixtures/posts_in_subfolder/posts");
+
+    assert_eq!(starts_with_path(Path::new("tests/fixtures/posts_in_subfolder"), posts_folder),
+               false);
```

Comment:
You can leave out true and false in this and the following calls and replace `assert_eq!` with `assert!`